.docker/entrypoint/lib/extensions/object_extensions.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require_relative '../errors/file_not_found_error'
+require_relative '../errors/directory_not_found_error'
+
+# Object extensions
+class Object
+  def valid_array?
+    !nil? && is_a?(Array) && !empty?
+  end
+
+  def must_be_a_directory!
+    must_be_a! :non_empty, String
+
+    dnfe = Jekyll::PlantUml::DirectoryNotFoundError
+    raise dnfe, "#{self} does not exist" unless Dir.exist?(self)
+  end
+
+  def must_be_a_file!
+    must_be_a! :non_empty, String
+
+    fnfe = Jekyll::PlantUml::FileNotFoundError
+    raise fnfe, "#{self} cannot be found." unless writable_file?
+  end
+
+  def writable_file?
+    return true if File.writable? self
+
+    false
+  end
+
+  def value_for(key)
+    must_be_a! Hash
+    raise ArgumentError, 'Hash cannot be empty' if empty?
+    raise ArgumentError, "No '#{key}' key found in the hash" unless key? key
+
+    self[key]
+  end
+
+  def must_be_a!(*args)
+    parsed_args = parse_args(*args)
+    type = parsed_args[:type]
+    specifier = parsed_args[:specifier]
+    raise ArgumentError, 'Type cannot be nil' if type.nil?
+    raise ArgumentError, "#{type} cannot be nil" if nil?
+    raise ArgumentError, "#{self.class} is not a #{type}" unless is_a? type
+    raise ArgumentError, "#{self.class} cannot be empty" if specifier == :non_empty && empty?
```

Comment:
Codacy found an issue: [Modifier form of `if` makes the line too long.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)